Pipelines/Templates/tests.yaml

```diff
@@ -78,50 +45,49 @@ steps:
       Stop-Job $ljob
       Remove-Job $ljob
       Stop-Process $proc
-    }
-    finally {
-      Write-Output '====================================================='
-      Write-Output '               PlayMode Tests Finished               '
-      Write-Output '====================================================='
-
-      if (Test-Path $logFile) {
-        Write-Output '====================================================='
-        Write-Output '             Begin PlayMode Tests Logs               '
-        Write-Output '====================================================='
-
-        Get-Content $logFile
-
-        Write-Output '====================================================='
-        Write-Output '              End PlayMode Tests Logs                '
-        Write-Output '====================================================='
-      }
-      else {
-        Write-Output 'PlayMode Tests log missing!'
       }
-
-      if (Test-Path $resultFile) {
+      finally {
         Write-Output '====================================================='
-        Write-Output '            Begin PlayMode Tests Results             '
-        Write-Output '====================================================='
-
-        Get-Content $resultFile
-
-        Write-Output '====================================================='
-        Write-Output '             End PlayMode Tests Results              '
+        Write-Output '               PlayMode Tests Finished               '
         Write-Output '====================================================='
+        if (Test-Path $logFile) {
+          Write-Output '====================================================='
+          Write-Output '             Begin PlayMode Tests Logs               '
+          Write-Output '====================================================='
+          Get-Content $logFile
+          Write-Output '====================================================='
+          Write-Output '              End PlayMode Tests Logs                '
+          Write-Output '====================================================='
+        }
+        else {
+          Write-Output 'PlayMode Tests log missing!'
+        }
+        if (Test-Path $resultFile) {
+          Write-Output '====================================================='
+          Write-Output '            Begin PlayMode Tests Results             '
+          Write-Output '====================================================='
+          Get-Content $resultFile
+          Write-Output '====================================================='
+          Write-Output '             End PlayMode Tests Results              '
+          Write-Output '====================================================='
+        }
+        else {
+          Write-Output 'PlayMode Tests results missing!'
+        }
+      Write-Output $($proc.ExitCode)
```

Comment:
Can this also be reverted to the previous printout? It provided better visibility into what was being printed by labeling it "Exit code"

Reply:
Was having issues with that at one point, I'll revert it and make sure there aren't any issues